### PR TITLE
MM-16264 - add cypress-test-data command to make file

### PR DESCRIPTION
**Pull request by thekiiingbob**
#### Summary
Added a new command in the make file for `cypress-test-data` to increase max users per team as well as create 60 users vs our default of 15.

#### Ticket Link
[JIRA](https://mattermost.atlassian.net/browse/MM-16264)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -444,6 +444,15 @@ test-data: start-docker ## Add test data to the local instance.
...
 	@echo ========================================================================
 
+cypress-test-data: start-docker ## Add test data to the local instance.
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) config set TeamSettings.MaxUsersPerTeam 100
+	$(GO) run $(GOFLAGS) -ldflags '$(LDFLAGS)' $(PLATFORM_FILES) sampledata -w 4 -u 60 
+	@echo You may need to restart the Mattermost server before using the following
+	@echo ========================================================================
+	@echo Login with a system admin account username=sysadmin password=sysadmin
+	@echo Login with a regular account username=user-1 password=user-1
+	@echo ========================================================================
```
Comment (saturninoabril) [inline Makefile]: Looks good though it feels redundant to me. 0/5 but what if we just modify the existing `test-data`?  I don't see any issue of doing so, probably longer time to generate test data depending on local machine (but actually when I tried, there seemed to be no difference at all).

Comment (lieut-data) [reply Makefile]: Agree with @saturninoabril -- unless there's a compelling reason to deviate here, wonder if we can reduce the `Makefile` targets.

Comment (thekiiingbob) [reply Makefile]: Works for me, I'll throw it in the primary test data command.

**Comment by saturninoabril** [comment]
@thekiiingbob Please merge master into this branch.

**Pull request merged**